### ISSUE TITLE
Warrior Charge ability Issue

### ISSUE THREAD

**Issue by ghostwipe** [dalaranwow/dalaran-wow#4432]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Gnome Warrior. Globally, when using my charge ability to attack a mob, I am sent to the middle of the mob, basically "inside" it's hit box. When I attempt to attack I get a "Facing the wrong direction" notification. I then have to back up a step in order to start attacking.

**Expected behaviour**: After charging, I should be placed at a suitable distance to immediately use an offensive ability. 

**Steps to reproduce the problem**:

1. Charge at mob.
2. Get to "middle" of mob.
3. Try to attack and get notification.

**Include proofs for this behaviour**
Not sure what this means, but found similar thread on wowhead.com

http://www.wowhead.com/forums&topic=134318.2/positioning-issues-while-tanking

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/3655, the fix needs to be tested on PTR.
